promptsource/templates/xcopa/id/templates.yaml

```
@@ -0,0 +1,211 @@
+dataset: xcopa
+subset: id
+templates:
+  1a87b487-1570-4873-aed9-b84d2fc0476c: !Template
+    answer_choices: '{{choice1}} ||| {{choice2}}'
+    id: 1a87b487-1570-4873-aed9-b84d2fc0476c
+    jinja: "{{ premise }} \n\nI am hesitating between two options. Help me choose\
+      \ the more likely {% if question == \"cause\" %} cause: {% else %} effect: {%\
+      \ endif %}\n- {{choice1}}\n- {{choice2}} ||| {% if label != -1 %}{{ answer_choices[label]\
+      \ }}{%endif%}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      languages:
+      - en
+      metrics:
+      - Accuracy
+      original_task: true
+    name: i_am_hesitating
+    reference: ''
+  336c4c72-40e3-4122-881e-8cd7a1881eec: !Template
+    answer_choices: '{{choice1}} ||| {{choice2}}'
+    id: 336c4c72-40e3-4122-881e-8cd7a1881eec
+    jinja: "{% if question == \"cause\" %} \n{{ premise }} Why? \"{{ answer_choices[0]\
+      \ }}\" or \"{{ answer_choices[1] }}\"? ||| {% if label != -1 %}{{ answer_choices[label]\
+      \ }}{%endif%}\n{% endif %}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      languages:
+      - en
+      metrics:
+      - Accuracy
+      original_task: true
+    name: "\u2026why? C1 or C2"
+    reference: ''
+  482f0b87-e748-4e98-8cc8-a23386bc50c3: !Template
+    answer_choices: '{{choice1}} ||| {{choice2}}'
+    id: 482f0b87-e748-4e98-8cc8-a23386bc50c3
+    jinja: "{{ premise }} \n\nWhat's the best option?\n- {{choice1}}\n- {{choice2}}\n\
+      \nWe are looking for {% if question == \"cause\" %} a cause {% else %} an effect\
```

Else it produces a double space
```suggestion
      \nWe are looking for {% if question == \"cause\" %}a cause {% else %}an effect\
```